src/m_menu.c

```diff
@@ -7005,18 +7005,18 @@ void M_Init(void)
   // [crispy] remove DOS reference from the game quit confirmation dialogs
   {
     const char *platform = SDL_GetPlatform();
-    const char *string;
```

Comment:
So, the OS replacement gimmicks still work?

Reply:
Yes:
![image](https://user-images.githubusercontent.com/5077629/231671978-4bdca81f-b906-4604-9f5c-9b1e3ebe6ff1.png)